association_backend/association/domain/entities/subscriptions.py

```diff
@@ -0,0 +1,80 @@
+from dataclasses import dataclass
+from datetime import datetime
+from enum import Enum
+from typing import Optional
+
+import pydantic
+from sqlalchemy import Column, DateTime, ForeignKey, Integer, String, Table
+from sqlalchemy.orm import registry, relationship
+
+
+class UserData(pydantic.BaseModel):
+    email: str
+    user_id: int
+
+
+class SubscriptionState(str, Enum):
+    PENDING = "pending"
+    ACTIVE = "active"
+    EXPIRED = "expired"
+    FIRST_PAYMENT_EXPIRED = "not-created"
```

Comment:
```suggestion
    FIRST_PAYMENT_EXPIRED = "first-payment-expired"
```

what does first payment expired mean?